tools/__tasks__/compile/javascript/webpack.js

```diff
@@ -1,4 +1,34 @@
+// eslint-disable-next-line import/no-unassigned-import
+require('any-observable/register/rxjs-all');
```

Comment:
tbh, i don't really know :) i guess its an implementation of an observer, and any observer just uses it? copy and pasted!